content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -462,6 +462,8 @@ kubectl get clusterroles system:discovery -o yaml
 
 NOTE: editing the role is not recommended as changes will be overwritten on API server restart via auto-reconciliation (see above).
 
+As of Kubernetes version 1.14+, the default RBAC discovery ClusterRoles are:
```

Comment:
suggest moving version-specific notes into `Description` as "Prior to 1.14, this role was also bound by default to `system:authenticated`" or "This role exists as of 1.14"

Reply:
done